scipy/optimize/_lsq/least_squares.py

```diff
@@ -234,45 +234,40 @@ def least_squares(
         kwargs={}):
     """Solve a nonlinear least-squares problem with bounds on the variables.
 
-    Let f(x) be an m-dimensional function of n variables and rho(s) be a
-    scalar function, `least_squares` finds a local minimum of::
+    Given f(x), an m-dimensional function of n variables (the residuals),
+    and rho(s), a scalar function (the loss function), `least_squares`
+    finds a local minimum of the cost function F(x)::
 
```

Comment:
Nitpick: I would tend to read `m-dimensional function of n variables (the residuals)` as if these n variables are themselves residuals.